### DISCUSSION TITLE
API - How to get URL of a Page based on labelAdded webhook

### DISCUSSION THREAD

**Discussion by yunanwg** [omnivore-app/omnivore#3434] [Q&A]
Hi,

I was wondering if I can get some ideas about how to query an URL of a Page based on the returned data of `labelAdded` webhook.

As a newbie to GraphQL, I cannot find a concrete method to search a Page based on its ID. The `query` type has only `article` and `search` parameters and either of them seems not to be able to intake a ID argument.

The idea of this workflow is that while a label `To Bookmark` is added in Omnivore, the triggered webhook will trigger a N8N workflow that add the link to Linkding, a bookmarking service.

Thanks a lot!

**Comment by ssejourne** [comment]
Hi, 

more or less the same use case... I would like to use the webhook to trigger a n8n workflow, and I may need at least the original URL and idealy the content to process it for my "_Second Brain_".

If someone is able to drop some simple exemples (based on Curl?) it would be very useful as the documentation of the API need more details

Many thanks !
